pom.xml

```diff
@@ -137,6 +137,7 @@
         <apache.io.version>2.6</apache.io.version>
         <io.confluent.ksql.version>6.2.0-0</io.confluent.ksql.version>
         <io.confluent.schema-registry.version>${confluent.version.range}</io.confluent.schema-registry.version>
+        <kafka.version>6.1.0-beta201006024150</kafka.version>
```

Comment:
I thought we were trying to use a hotfix version for this release?

Reply:
Yes, you are right. I will fix it